sdk/src/test/tutorial/java/parity/FeatureLookthroughTest.java

```diff
@@ -0,0 +1,19 @@
+package parity;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+public class FeatureLookthroughTest {
+
+    @Test
+    public void getAnnotationsAndWriteToFile() throws ClassNotFoundException, IOException, URISyntaxException {
```

Comment:
Good naming of the test, but anytime you have "and" in there is means you're probably mixing concerns in the test. This should be 2 tests 
1) create a set of test classes specific for this test and validate the annotations are found as expected
2) validate that file writer takes a list of features and writes to the file as expected